src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3,6 +3,7 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Diagnostics.Contracts;
```

Comment:
>System.Diagnostics.Contracts [](start = 6, length = 28)

Can be removed?

Reply:
I use the contracts in some of the new code i wrote.